src/python/pants/engine/isolated_process.py

```diff
@@ -27,6 +27,7 @@ class ExecuteProcessRequest(datatype([
   ('output_directories', hashable_string_list),
   # NB: timeout_seconds covers the whole remote operation including queuing and setup.
   ('timeout_seconds', Exactly(float, int)),
+  ('local_only_scratch_files', Digest),
```

Comment:
I was serious about the rename; please call this `unsafe_local_only_files_because_we_favor_speed_over_correctness_for_this_rule`

I would be happy using `incorrect` or `buggy` or `dangerous` instead of `unsafe`, if you prefer.

Reply:
Renamed. Thanks!